invenio_opendata/base/recordext/fields/opendata.cfg

```diff
@@ -203,21 +203,37 @@ action_note:
     producer:
         json_for_marc(), {"583__a": "action", "583__u": "url", "583__y": "description", "583__w": "recid"}
 
+# 593__a 593__b
+
+local_notes:
```

Comment:
Since we have `documentation_note` (singular), please use singular also here (`local_note`), for consistency reasons.